lib/livebook/application.ex

```diff
@@ -179,7 +187,7 @@ defmodule Livebook.Application do
   end
 
   defp display_startup_info() do
-    if Phoenix.Endpoint.server?(:livebook, LivebookWeb.Endpoint) do
+    if not serverless?() and Phoenix.Endpoint.server?(:livebook, LivebookWeb.Endpoint) do
```

Comment:
I would say this one is worth being Process.whereis(...), because we should never log if it is not running, but your call. :)